### PR TITLE
[Docs] Overview

### PR DESCRIPTION
**Pull request by akwong2**
### Change Logs
1. Setting up the library
2. Recommendations

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,48 @@
...
+### Retrieving an API-Key
+You need a valid API-key, which you can get for free at  [https://www.breinify.com](https://www.breinify.com/) . In the examples, we assume you have the following API key:
+**938D-3120-64DD-413F-BB55-6573-90CE-473A**
+It is recommended to use signed messages when utilizing the react library. A signed messages ensures, that the request is authorized. To activate signed message ensure that `Verification Signature` is enabled for your key (see  [Breinify’s API Docs](https://docs.breinify.com/#request-an-api-key)  for further information). In this documentation we assume that the following secret is attached to the API key and used to sign a message.
+**utakxp7sm6weo5gvk7cytw==**
+
+### Installing
+The library is available on  [npm](https://www.npmjs.com/package/breinify-node)  and can be added using:
+```bash
+npm i brein-react-api-library-react --save
```
Comment (timothyyue-b) [inline README.md:26]: We should check if its a dev-dependency or an actual dependency. I'm not sure at this point and time, but something to keep in mind.

File: docs/recommendations.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
+# Recommendations
+Retrieves a list of recommended products for the user supplied in the user object of the body. To use the recommendations components, we’ve assumed that you’ve set up [BreinifySetup](/README.md#configuring-the-library).
+
+## Hooks
+### useRecommendations
```
Comment (timothyyue-b) [inline docs/recommendations.md:5]: I think we should add a description here on what this hook does. For example, 
```
Provides a standard API wrapper that calls the `getRecommendation` fetch method
```

File: docs/recommendations.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+            {isFailure && <>{JSON.stringify(error, null, 2)}</>}
+            <button onClick={onButton}>Click Here</button>
+        </div>
+    );
+}
+```
+
+## Calls
+### getRecommendations
+If you don’t want to use the hook (`useRecommendations`) we provided, we also exposed `getRecommendations`  to help call our API.
```
Comment (timothyyue-b) [inline docs/recommendations.md:53]: I think we can exclude the `If you don't want to use the hook` section, we should phrase it more like `We've also exposed the lower level utility that aids in calling our recommendation API`

File: docs/recommendations.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+            {isSuccess && <>{JSON.stringify(data, null, 2)}</>}
+            {isFailure && <>{JSON.stringify(error, null, 2)}</>}
+            <button onClick={onButton}>Click Here</button>
+        </div>
+    );
+}
+```
+
+## Calls
+### getRecommendations
```
Comment (timothyyue-b) [inline docs/recommendations.md:52]: Somewhere in the doc, we should document the `return` type of `getRecommendations`

**Review by timothyyue-b — APPROVED** [review]
Initial docs look good! I left some inline comments, which is some things I noticed while reading through the document. I think some of the wording can be changed, but this is something that we could sit down and discuss

**Comment by timothyyue-b** [comment]
Also, reading the code from #14 , I noticed we didn't document the `selector` function in the `getRecs` utility function either ( the second parameter of `getRecs` )

File: README.md [hunk 5]
```diff
@@ -1,2 +1,48 @@
...
+* [API](#api)
+* [Further Links](#further-links)
+<!-- TOC -->
+
+## Overview
+Breinify’s React API library provides hooks, components, and type definitions to call Breinify’s API.
+
+## Getting Started
+### Retrieving an API-Key
+You need a valid API-key, which you can get for free at  [https://www.breinify.com](https://www.breinify.com/) . In the examples, we assume you have the following API key:
```
[the remark this answers is not in the document]
Comment (akwong2) [reply README.md:18]: According to the docs it is 😄: https://github.com/Breinify/brein-api-library-javascript-browser#retrieving-an-api-key
I'll change the URL

[the remark this answers is not in the document]
Comment (timothyyue-b) [reply README.md]: Screen Shot 2022-08-02 at 2 57 55 PM

According to README.md last updated, this information is 5 years old 🤣

**Comment by akwong2** [comment]
> Also, reading the code from #14 , I noticed we didn't document the `selector` function in the `getRecs` utility function either ( the second parameter of `getRecs` )

Solved in [a357420](https://github.com/Breinify/brein-api-library-react/pull/13/commits/a357420bd7ad01a4a43ca17063080d33f66dfc1b)

File: README.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (akwong2) [reply README.md]: hmm good point, we'll ask Philipp in the meeting

**Pull request merged**